server/neptune/workflows/activities/terraform.go

```diff
@@ -98,10 +99,33 @@ func NewTerraformActivities(
 	}
 }
 
+func getEnvs(directEnvs map[string]string, dynamicEnvs []EnvVar) (map[string]string, error) {
+	envs := make(map[string]string)
+
+	for k, v := range directEnvs {
+		envs[k] = v
+	}
+
+	for _, e := range dynamicEnvs {
+		v, err := e.GetValue()
+
+		if err != nil {
```

Comment:
nit: remove space(s) between invocation and catching of errors. Will only comment once but there are several throughout.

Reply:
ooo yeah i tend to leave a lot of spacing in my code out of personal preference but if that's not the overall preference I can remove.

Reply:
Yeah mostly just the err handling should be close to the invoking code without space, most style guides will advocate for this as well as the go std lib's.